.travis.yml

```diff
@@ -43,7 +43,7 @@ compiler:
 env:
   global:
     - MAKEFLAGS="-j2"
-    - LLVM_VERSION=13
+    - LLVM_VERSION=12
```

Comment:
Looking at the travis failure, it seems like you also need to update "llvm-toolchain-bionic" to "llvm-toolchain-bionic-12" on line 28 above.

Reply:
Right, thanks! I have updated the patch.